source/includes/_email_survey.md

```diff
@@ -50,6 +50,7 @@ curl "https://api.wootric.com/v1/email_survey" \
   -d "emails[]=john@example.com" \
   -d "emails[]=martin@example.com" \
   -d "survey_immediately=true" \
+  -d "survey_settings[custom_messages][logo_url]=https://company_logo_url.com" \
```

Comment:
@marinhero it should be `survey_settings[logo_url]=https://company_logo_url.com`

Reply:
My deepest apologies, it's fixed now.